cost-analyzer/templates/_helpers.tpl

```diff
@@ -93,6 +93,39 @@ Kubecost 2.0 preconditions
 
 {{- end -}}
 
+{{- define "cloudIntegrationFromProductConfigs" }}
+  {
+    "aws": [
+      {
+          "athenaBucketName": "{{ .Values.kubecostProductConfigs.athenaBucketName }}",
+          "athenaRegion": "{{ .Values.kubecostProductConfigs.athenaRegion }}",
+          "athenaDatabase": "{{ .Values.kubecostProductConfigs.athenaDatabase }}",
+          "athenaTable": "{{ .Values.kubecostProductConfigs.athenaTable }}",
+          "projectID": "{{ .Values.kubecostProductConfigs.athenaProjectID }}"
+          {{- if and ((.Values.kubecostProductConfigs).awsServiceKeyName) ((.Values.kubecostProductConfigs).awsServiceKeyPassword) }},
+          "serviceKeyName": "{{ .Values.kubecostProductConfigs.awsServiceKeyName }}",
+          "serviceKeySecret": "{{ .Values.kubecostProductConfigs.awsServiceKeyPassword }}"
+          {{- end }}
+      }
+    ]
+  }
+{{- end }}
```

Comment:
@Sean-Holcomb Is this still a valid way of setting up your cloud integration? I've been told from a user that the only supported method now is view `cloud-integration.json`.

Reply:
This creates that secret via helm, fixing that problem.

We also fixed the UI to only show status of cloud-integration and not the old API.

Reply:
@jessegoodier Awesome! We'll do the same for Azure & GCP in a separate PR?